### PR TITLE
[WIP] Adapt iscsi_server to SLE15

### PR DESCRIPTION
**Pull request by DrMullings**
Adapted shortscuts to sle15

- Related ticket: https://progress.opensuse.org/issues/28276
- Needles: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/758
- Verification run: WIP

### CODE REVIEW

**Comment by okurz** [comment]
+1, except WIP label

File: lib/version_utils.pm [hunk 1]
```diff
@@ -289,5 +289,6 @@ sub is_system_upgrading {
 }
 
 sub is_pre_15 {
-    return (is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0')) || !is_tumbleweed;
+    return 0 if is_tumbleweed;
+    return is_sle('<15') || is_leap('<15.0') ;
```
Comment (rwx788) [inline lib/version_utils.pm]: is_leap doesn't support new functionality we have for is_sle. Still have to use leap_version_at_least here

Comment (DrMullings) [reply lib/version_utils.pm]: It was implemented last week, also I asked you in the ticket, but I have to re-run the verification to show you the behaviour

**Comment by DrMullings** [comment]
@rwx788: 
http://pinky.arch.suse.de/tests/469
Is this due to the lack of multimachine setup, a bug in the product or do I need to adapt the test code?

**Comment by rwx788** [comment]
@DrMullings I guess this part should still work without multimachine setup, but you will need it. Have you filed a bug regarding typos in the error message?

**Comment by okurz** [comment]
please mention the SLE needles MR. It has been merged alreay

**Comment by DrMullings** [comment]
Blocked by
https://bugzilla.suse.com/show_bug.cgi?id=1085421
Imho we should merge this to monitor any progress on OSD

**Pull request closed without merging**